### ISSUE TITLE
Log4j2PluginsCacheFileTransformer not working correctly?

### ISSUE THREAD

**Issue by dominicE** [johnrengelman/shadow#410]
Using 4.0.0 (4.0.1 not working with shadow-transformer from Boegl) of shadow with latest shadow-transformer from Sebastian Boegl works fine. Using latest Log4j (2.11.1).

### Shadow Version
latest - 4.0.1

### Gradle Version
latest - 4.10.2

### Expected Behavior
fatJar should launch with Log4j2 config file. Running from within IDE (intelliJ) works fine

### Actual Behavior
fatJar launches with error messages:

```
ERROR StatusLogger Unrecognized format specifier [d]
ERROR StatusLogger Unrecognized conversion specifier [d] starting at position 16 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [thread]
```

### Gradle Build Script(s)
```
shadowJar {
    transform(Log4j2PluginsCacheFileTransformer)
    transform(ServiceFileTransformer) {
        path = 'META-INF/services'
        include 'org.eclipse.jetty.http.HttpFieldPreEncoder'
    }
    
    manifest{
        attributes 'Class-Path': 'libs/ext'
    }

    dependencies {
        // exclude because install4j provides its dependency
        exclude(dependency('install4j-runtime:.*'))
    }
}
```

### Content of Shadow JAR (`jar tf <jar file>` - post link to GIST if too long)
https://gist.github.com/dominicE/be8936395b2881139dfbb7f8de8d59ee

**Comment by johnrengelman** [comment]
4.0.1 has a breaking API change for transformers per the change log (yeah I know that’s not good form, but there were features for the 4.0.0 release that were broken and couldn’t be fixed without the change)

**Comment by johnrengelman** [comment]
I’m not clear from this issue if you are using the built in log4jtranaformer that is included in shadow starting in 4.0.0

**Comment by johnrengelman** [comment]
Closing as this transformer is now embedded in Shadow and the external dependency is no longer needed.
